cmd/validate_test.go

```diff
@@ -43,11 +43,11 @@ func TestIsYAML(t *testing.T) {
 	assert.False(t, safeguards.IsYAML(fileNotYaml))
 	assert.True(t, safeguards.IsYAML(fileYaml))
 
-	manifestFiles, err := safeguards.GetManifestFiles(dirNotYaml)
+	manifestFiles, err := preprocessing.GetManifestFiles(dirNotYaml)
```

Comment:
you should move the whole `TestIsYAML` function to the preprocessing package - validate_test doesn't need to test a different package